minicluster/src/main/java/alluxio/master/AbstractLocalAlluxioCluster.java

```diff
@@ -256,9 +267,10 @@ protected void setupTest() throws IOException {
    * @throws Exception when the operation fails
    */
   public void stop() throws Exception {
-    stopFS();
-    stopUFS();
-
+    if (mStarted) {
```

Comment:
ideally `stopFS` and `stopUFS` should be a no-op if the cluster has not been started ... then we do not need to keep a boolean flag about whether it has been started